examples/basic-tracer-node/index.js

```diff
@@ -22,13 +22,13 @@ if (EXPORTER.toLowerCase().startsWith('z')) {
   exporter = new CollectorExporter(options);
 }
 
-const registry = new BasicTracerRegistry();
+const provider = new BasicTracerProvider();
 
 // Configure span processor to send spans to the provided exporter
-registry.addSpanProcessor(new SimpleSpanProcessor(exporter));
+provider.addSpanProcessor(new SimpleSpanProcessor(exporter));
 
-// Initialize the OpenTelemetry APIs to use the BasicTracerRegistry bindings
-opentelemetry.initGlobalTracerRegistry(registry);
+// Initialize the OpenTelemetry APIs to use the BasicTracerProvider bindings
+opentelemetry.initGlobalTracerProvider(provider);
```

Comment:
Same here, opentelemetry should come from api instead of core.

Reply:
Also, looks like it is safe to remove `@opentelemetry/core` from dependencies, once you do above change.

Reply:
Fixed in all examples